torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4659,9 +4685,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestCommon', 'test_variant_consistency_eager'),),
            sample_inputs_func=sample_inputs_addcmul_addcdiv),
     OpInfo('amax',
+           ref=lambda a, dim=None, keepdim=False, **kwargs: np.amax(a, axis=dim, keepdims=keepdim, **kwargs),
```

Comment:
- NumPy takes `keepdims` as an argument while PyTorch takes `keepdim`.
- From the above comment (todo: update link), we pass `keepdim` as keyword argument but since NumPy doesn't have it, hence wrapping the function in a lambda and passing `keepdim` value to `keepdims` is recommended solution.

Reply:
This is correct. References should have the same signatures as the PyTorch functions (when required).

Separately, in the future we should update our arg parsing to accept both keepdim and keepdims. I'll file some tracking issues.